### PR TITLE
Hebrew translation

### PR DESCRIPTION
**Pull request by nirgn975**
I've translated the 2.0.0 spec to Hebrew (I'm a native Hebrew speaker).

### CODE REVIEW

**Comment by haacked** [comment]
Excellent! I have no good way of reviewing this. Do you know someone who could proofread it?

**Comment by nirgn975** [comment]
Maybe @RoySegall can?

**Comment by RoySegall** [comment]
Yeah, I'll review it once I'll manage to install and run it.

**Comment by haacked** [comment]
@RoySegall It's a translation of existing content so it's not strictly necessary to install and run it to review it. You can just put inline comments here: https://github.com/mojombo/semver.org/pull/164/files

But if you want to run it, by all means! :smile:

**Comment by RoySegall** [comment]
I needed to align the text to the right, but I changed it through the developer tools in chrome. I'll give a second look tonight.

**Comment by nirgn975** [comment]
@RoySegall The text should be align to the right, apart from the FAQ questions, they do not (I think it's because they are in `###`, but it's the same as in another RTL languages (see `ar`)).

**Comment by RoySegall** [comment]
But it's aligned to the right on Github. Because GitHub is not, and thanks god for that, is not a multilingual site.

File: lang/he/index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+==============================
+
+סיכום
+-------
+
+בהינתן מספר גרסה MAJOR.MINOR.PATCH, העלה את:
+
+1. גרסה ראשית (MAJOR) כאשר אתה עושה שינויים שאינם עולים בקנה אחד עם ה API,
+1. גרסה משנית (MINOR) כשאר אתה מוסיף פונקציונאליות בצורה שמותאמת אחורה, ו
+1. גרסת תיקון (PATCH) כאשר אתה מבצע תיקוני באגים שמותאמים אחורה.
```
Comment (RoySegall) [inline lang/he/index.md:17]:
```
1.
1.
1.
```
Shouldn't this be 1,2,3?

File: lang/he/index.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+בתוכנות עם המון תלויות, שחרור גרסה של חבילה חדשה יכול במהירות להפוך לסיוט. אם מפרט התלויות הדוק מידי, אתה בסכנה של ״נעילת גרסה (version lock)״ (חוסר היכולת לשדרג חבילה מבלי לשחרר גרסה חדשה של כל חבילה תלויה). אם מפרט התלויות רופף מידי, באופן בלתי נמנע אתה תכווה ע״י ״מתירנות גרסה (version promiscuity)״ (השערה של תאימות, יותר מהסביר, עם גרסאות עתידיות). ״גיהנום התלויות״ הוא כאשר אתה נמצא במקום בו נעילת גרסה ו/או מתירנות גרסה מונעים ממך בקלות ובבטחה להניע את הפרויקט שלך קדימה.
+
+כפתרון לבעיה זו, אני מציע סט פשוט של כללים ודרישות המכתיבים כיצד מספרי גרסאות יוקצו ויתווספו. כללים אלה מבוססים על מנהגים משותפים, קיימים ונפוצים בפרויקטי קוד פתוח וסגור. כדי שמערכת זו תפעל, תחילה צריך להכריז של API ציבורי. זה אולי יהיה מורכב מתיעוד או יאכף על ידי הקוד עצמו. בכל מקרה, זה חשוב שה API יהיה ברור ומדויק. ברגע שאתה מגדיר את ה API הציבורי שלך, אתה מתקשר בו את השינויים עם תוספות ספצפיות למספר הגרסה. הסתכל על תבנית גרסה של X.Y.Z (בפורמט Major.Minor.Patch). תיקוני באגים שלא משפיעים על ה API יגדילו את מספר את מספר התיקון (Patch), תוספות / שינויים תואמי API לאחור יגדילו את הגרסה המשנית (Minor), ושינויים שאינם נתמכים לאחור ב API יגדילו את הגרסה הראשית (Major).
```
Comment (RoySegall) [inline lang/he/index.md]: כפתרון => על מנת לפתור בעיה זו

Comment (RoySegall) [reply lang/he/index.md]: כדי => על מנת

Comment (RoySegall) [reply lang/he/index.md]: תחילה צריך להכריז של API ציבורי => צריך תחילה להכריז על API ציבורי.

File: lang/he/index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+אני קורא למערכת  ״גירסאות סמנטיות". במסגרת זו, מספרי גרסאות, והדרך שבה הם משתנים מעבירה משמעות על הקוד ועל מה שהשתנה מגרסה אחת לאחרת.
```
Comment (RoySegall) [inline lang/he/index.md]: מספרי גרסאות, והדרך => מספרי גרסאות והדרך
אין פסיק לנפי ו חיבור

File: lang/he/index.md (hunk 1 above)
Comment (RoySegall) [reply lang/he/index.md:17]: It's also appear in other line numbering.

File: lang/he/index.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+
+מפרט גירסאות סמנטיות (SemVer)
+------------------------------------------
+
+מילות המפתח ״חייב (MUST)״, ״אסור (MUST NOT)״, "חובה (REQUIRED)״, ״תהא (SHELL)״, ״לא (SHELL NOT)״, ״צריך (SHOULD)״, ״לא צריך (SHOULD NOT)״, ״מומלץ (RECOMMENDED)״, ״יכול (MAY)״, ״אפשרי (OPTIONAL)״ במסמך זה צריכות להיות מפורשות כמתואר במסמך [RFC 2119](http://tools.ietf.org/html/rfc2119).
```
Comment (RoySegall) [inline lang/he/index.md]: ״יכול (MAY)״, ״אפשרי (OPTIONAL)״ במסמך זה צריכות להיות =>  ״יכול (MAY)״ ו-״אפשרי (OPTIONAL)״ אשר מופיעות במסמך, צריכות להיות

File: lang/he/index.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+1. תוכנה באמצעות גירסאות סמנטיות נדרשת להכריז על API ציבורי. ניתן להכריז על API זה בקוד עצמו או אך ורק בתיעוד. עם זאת זה צריך להיעשות, להיות מדויק ומקיף.
```
Comment (RoySegall) [inline lang/he/index.md]: להיעשות, להיות מדויק ומקיף => להיעשות מדויק ומקיף.

File: lang/he/index.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+1. מספר גרסה רגילה חייב (MUST) להיות בצורה X.Y.Z כאשר Y, X, ו Z הם מספרים שלמים (Integers) שאינם שליליים, ואסור (Must Not) שיכילו אפסים מובילים. X הוא גרסה ראשית (Major), לאחריו Y הוא גרסה משנית (Minor), ולבסוף Z הוא גרסת תיקון (Patch).
+כל רכיב במספר הגרסה חייב (MUST) להגדיל את המספר נומרית. לדוגמה: 1.9.0 -> 1.10.0 -> 1.11.0.
+
+1. לאחר שחבילה עם מספר גרסה כבר פורסמה, אסור (MUST NOT) לשנות את התוכן של החבילה באותה גרסה. כל שינוי חייב (MUST) להשתחרר כגרסה חדשה.
```
Comment (RoySegall) [inline lang/he/index.md]: לאחר שחבילה עם מספר גרסה כבר פורסמה => לאחר שחבילה שוחררה

File: lang/he/index.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+1. גרסה ראשית אפס (0.z.y) היא עבור פיתוח ראשוני. כל דבר יכול להשתנות בכל זמן. ה API הציבורי לא צריך להיחשב יציב.
```
Comment (RoySegall) [inline lang/he/index.md]: כל דבר יכול להשתנות בכל זמן. ה API הציבורי לא צריך להיחשב יציב. => בהינתן שכל דבר יכול להשתנות בכל זמן, ה-API הציבורי לא צריך להיחשב יציב.

File: lang/he/index.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+
+1. גרסה 1.0.0 מגדירה את ה API הציבורי. הדרך שבה מספר הגרסה עולה לאחר שחרור גרסה זו תלוי ב API הציבורי ובכיצד הוא משתנה.
```
Comment (RoySegall) [inline lang/he/index.md]: ב API הציבורי ובכיצד הוא משתנה. => ב-API הציבורי ובאופן בו הוא משתנה.

File: lang/he/index.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+
+1. גרסת תיקון (Patch) במיקום Z (כלומר x > 0 | Z.y.x) חייבת (MUST) לעלות אם ורק אם הוצגו תיקוני באגים תואמים לאחור. תיקון באג שתואם לאחור מוגדר כשינוי פנימי שמתקן התנהגות לא נכונה.
+
+1. גרסה המשנית (Minor) במיקום Y (כלומר x > 0 | z.Y.x) חייבת (MUST) לעלות שינויי פונקציונאליות חדש, תואם אחורה, הוצג ב API הציבורי. הוא חייב (MUST) לעלות אם פונקציונאליות כלשהי ב API הציבורי מסומן כ deprecated. הוא יכול (MAY) לעלות אם הוצגו שינויים או חידושיים משמעותיים בתוך הקוד הפרטי. הוא יכול (MAY) לכלול שינויים ברמת תיקון (Patch). מספר הגרסה ברמת תיקון (Patch) חייב (MUST) להיות מאופס ל0 כאשר מספר הגרסה המשנית (Minor) עולה.
+
```
Comment (RoySegall) [inline lang/he/index.md:52]: ל0 => ל-0
יש עוד מקום למטה שיש בו רווח לפני האפס.

File: lang/he/index.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+
+שאלות נפוצות
+---
+
+### כיצד עלי להתמודד עם תיקונים בשלב הפיתוח הראשוני `O.y.z`?
+
+הדבר הפשוט ביותר לעשות הוא להתחיל לשחרר את הפיתוח הראשוני שלך בגרסה 0.1.0 ואז להגדיל את הגרסה המשנית (Minor) לכל גרסה שבא אחריה.
+
+### איך אני יודע מתי לשחרר את גרסה `1.0.0`?
```
Comment (RoySegall) [inline lang/he/index.md]: איך אני יודע מתי => איך אדע מתי אוכל לשחרר את גרסא

**Comment by RoySegall** [comment]
I review this one. @Haacked you should learn Hebrew because we know English, so I think it's un fair for you: We know something that you don't ;)

Just to make sure - the spec.md and the index.md are the same, right? I don't need to review the second files because they look the same from the eagle eye view.

**Comment by haacked** [comment]
@RoySegall Thanks!

> you should learn Hebrew because we know English

ha! Fair. I wish I could learn all the languages for which we have Semver translations. Unfortunately I don't have a knack for human languages as much as I do for programming languages. And even those I struggle with. :smile:

So have all the changes been made? Is this good to go?

**Comment by haacked** [comment]
> Just to make sure - the spec.md and the index.md are the same, right? I don't need to review the second files because they look the same from the eagle eye view.

For now, yes. `index.md` and `spec/v2.0.0.md` should be exactly the same. Over time, they will diverge because `index.md` is always the latest _released_ version of SemVer. For example, once `v3.0.0` comes out, we'll add `spec/v3.0.0.md` and update `index.md` to the 3.0 version.

File: lang/he/index.md (hunk 1 above)
Comment (nirgn975) [reply lang/he/index.md:17]: Yes, I think it's github markdown flavor, but it's handle it. It'll output 
```
1. ...
2. ...
3. ...
```

All the other version write it this way, so I stay consistent

**Comment by nirgn975** [comment]
@RoySegall Thanks! I fixed everything but the line numbers.

@Haacked I think it's ready to be merged (:

**Comment by RoySegall** [comment]
About the line numbers - I don't think so. I looked in other languages and the markdown had 1, 2, 3...

But except that - Yalla, let's merge it.

**Comment by haacked** [comment]
Thanks for your help!

**Pull request merged**